Gemfile

```diff
@@ -36,6 +36,7 @@ gem 'nokogiri', '~> 1.10'
 gem 'nokogiri-happymapper', '~> 0.8.0', require: 'happymapper'
 gem 'resource_registry', git: 'https://github.com/ideacrew/resource_registry.git', branch: 'trunk'
 gem 'roo', '~> 2.1'
+gem 'roo-xls'
```

Comment:
Why do we need this new gem ?

Reply:
To read xls format files we need this otherwise it is throwing below error

`ArgumentError (Can't detect the type of /Users/rohith/workspace/with_git/quoting_tool/db/seedfiles/plan_xmls/me/xls_templates/service_areas/2023/c3ee968c-6831-49ed-aeea-dc4c5f3f7351_DATA_SERVICE_AREA.xls - please use the :extension option to declare its type.)`